modules/ROOT/pages/_partials/sync/server-auth.adoc

```diff
@@ -88,13 +88,13 @@ The `@hasRole` usage is as follows:
   * If the authenticated user has the role `admin` they will be authorized.
 * `@hasRole(role: [String])`
   * Example - `@hasRole(role: ["admin", "editor"])`
-  * If the authenticated user has **at least one of the roles** in the list, they will be authorized.
+  * If the authenticated user has at least one of the roles in the list, they will be authorized.
 
-**The default behaviour is to check client roles.** For example, `@hasRole(role: "admin"])` will check that user has a client role called `admin`. `@hasRole(role: "realm:admin"])` will check if that user has a realm role called `admin`
+The default behaviour is to check client roles. For example, `@hasRole(role: "admin"])` will check that user has a client role called `admin`. `@hasRole(role: "realm:admin"])` will check if that user has a realm role called `admin`
```

Comment:
`@hasRole(role: "admin"])` -> `@hasRole(role: "admin")`
`@hasRole(role: "realm:admin"])` -> `@hasRole(role: "realm:admin")`
This typo was also in the older version.